core-rust/core-api-server/src/core_api/handlers/lts/stream_account_transaction_outcomes.rs

```diff
@@ -37,7 +60,45 @@ fn handle_lts_stream_account_transaction_outcomes_internal(
         )));
     }
 
-    let _max_state_version = state_manager.store().max_state_version();
+    let query_account_change_index = state_manager.store().query_account_change_index();
+    if !query_account_change_index.is_enabled() {
+        return Err(server_error(
+            "This endpoint require AccountChangeIndex enabled.",
```

Comment:
I think we could provide a little more information here:

> This endpoint requires that the `AccountChangeIndex` is enabled on the node. To use this endpoint, you will need to enable the index, delete your database and resync.

Reply:
Agree more info is nice.

The database does not need to be deleted though (and node resynced).